<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KESAVAN-PC" ignored="0" name="Test" tests="4" failures="1" timestamp="2021-02-25T09:56:09 IST" time="6.775" errors="0">
  <testcase name="launchFB" time="0.015" classname="com.testNG.ParallelTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at com.testNG.ParallelTest.launchFB(ParallelTest.java:21)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchFB -->
  <testcase name="searchAcceessories" time="0.512" classname="com.testNG.TimeOutTest"/>
  <testcase name="searchTv" time="2.005" classname="com.testNG.TimeOutTest"/>
  <testcase name="launchGoogle" time="6.705" classname="com.testNG.ParallelTest"/>
</testsuite> <!-- Test -->
